manifests/config/apache.pp

```diff
@@ -225,9 +207,9 @@
       'no_proxy_uris' => $_proxy_no_proxy_uris,
       'path'          => pick($suburi, '/'),
       'url'           => $_proxy_backend,
-      'params'        => $_proxy_params,
+      'params'        => $proxy_params + { 'upgrade' => 'websocket' },
     },
-    'rewrites'            => $vhost_rewrites,
+    'rewrites'            => [],
```

Comment:
Do  we need this? I think we can drop it:
```suggestion
```

Reply:
let me check with the apache module and update